eng/common/README.md

```diff
@@ -19,5 +19,9 @@ The 'Sync eng/common directory' PRs will be created in the language repositories
 2. `azure-sdk-tools - sync - eng-common` is run automatically. It creates **Sync PRs** in each of the connected language repositories using the format `Sync eng/common directory with azure-sdk-tools for PR {Tools PR Number}`. Each **Sync PR** will contain a link back to the **Tools PR** that triggered it.
 3. More changes pushed to the **Tools PR**, will automatically triggered new pipeline runs in the respective **Sync PRs**. The **Sync PRs** are used to make sure the changes would not break any of the connected pipelines.
 4. Once satisfied with the changes;
-    - First merge all the **Sync PRs**. The **Tools PR** contains links to all the **Sync PRs**
-    - Finally merge the **Tools PR**. Each **Sync PR** contains the link to the corresponding **Tools PR**. 
\ No newline at end of file
+    - First make sure all checks in the **Sync PRs** are green and approved. The **Tools PR** contains links to all the **Sync PRs**.
```

Comment:
```suggestion
    - First make sure all checks in the **Sync PRs** are green and approved. The **Tools PR** contains links to all the **Sync PRs**. If for some reason the PRs is blocked by a CI gate get someone with permission to override and manually merge the PR. 
```

Reply:
I also think it might be worth adding a note about how to run the script locally to check the status of each of the sync PRs. I already added that support to the script.